main/src/mill/main/VisualizeModule.scala

```diff
@@ -43,7 +43,7 @@ trait VisualizeModule extends mill.define.TaskModule {
     )
     val visualizeThread = new java.lang.Thread(() =>
       while (true) {
-        val res = Result.create {
+        val res = Result.Success{
```

Comment:
This version does not catch exceptions and converts them into `Failing`. Is this on purpose?